AccessContextManager/synth.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import subprocess
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICBazel()
+common = gcp.CommonTemplates()
+
+library = gapic.php_library(
+    service='accesscontextmanager',
+    version='V1',
+    bazel_target='//google/identity/accesscontextmanager/v1:google-cloud-identity-accesscontextmanager-v1-php',
```

Comment:
Same here, the bazel target should be: 
```suggestion
    bazel_target='//google/identity/accesscontextmanager/v1:google-identity-accesscontextmanager-v1-php',
```

Reply:
It looks like it should be cloud:     name = "google-cloud-identity-accesscontextmanager-v1-php",